src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -53,26 +55,24 @@ public override int Run(TOptions analyzeOptions)
             return ((RuntimeErrors & RuntimeConditions.Fatal) == RuntimeConditions.NoErrors) ? SUCCESS : FAILURE;
         }
 
-        protected abstract void ValidateOptions(TContext context, TOptions analyzeOptions);
-
         private void Analyze(TOptions analyzeOptions, AggregatingLogger logger)
         {
             // 0. Log analysis initiation
             logger.AnalysisStarted();
 
-            // 1. Create our configuration property bag, which will be 
-            //    shared with all rules during analysis
-            PropertyBag policy = CreateConfigurationFromOptions(analyzeOptions);
-
-            // 2. Create context object to pass to skimmers. The logger
+            // 1. Create context object to pass to skimmers. The logger
```

Comment:
This change provoked a common issue with the framework: we require a certain amount of successful processing in order to log messages in a standard way. And so I reordered operations a bit (pushing off the 'policy' configuration to a later step).